scalameta/contrib/src/main/scala/scala/meta/contrib/instances/StatementsInstances.scala

```diff
@@ -0,0 +1,117 @@
+package scala.meta.contrib.instances
+
+import scala.collection.immutable.Seq
+import scala.meta._
+import scala.meta.contrib._
+import scala.meta.contrib.implicits.Statements
+
+trait StatementsInstances {
+  implicit val termBlockStats: Statements[Term.Block] = new Statements[Term.Block] {
+    override def getStats(block: Term.Block): Seq[Stat] = block.stats
+
+    override def replaceStats(block: Term.Block, newStats: Seq[Stat]): Term.Block =
+      block.copy(stats = newStats)
+  }
+
+  implicit val packageStats: Statements[Pkg] = new Statements[Pkg] {
+    override def getStats(pkg: Pkg): Seq[Stat] = pkg.stats
```

Comment:
I've changed my mind on this. I think we shouldn't hide any extra logic in this Statements type class. It should be focused on  extracting stats.